website/pages/en/index.js

```diff
@@ -113,90 +103,155 @@ class Contributors extends React.Component {
+              <div className="jest-card-logo" />
             </div>
+            <div className="jest-card-label jest-card-label-reverse">JEST</div>
+          </div>
+          <div className="jest-card-back">
+            <svg viewBox="0 0 200 200" style={{height: 150, width: 150}}>
+              <defs>
+                <path
+                  d="M100 100 m -75 0 a75 75 0 1 0 150 0 a 75 75 0 1 0 -150 0"
+                  id="runs-path"
+                />
+              </defs>
+              <circle
+                cx="100"
+                cy="100"
+                r="88"
+                stroke="#fff"
+                strokeWidth="8"
+                fill="#C2A813"
+              />
+              <g className="run-circle">
+                <circle cx="100" cy="100" r="50" fill="#fff" />
+                <circle
+                  cx="100"
+                  cy="100"
+                  r="45"
+                  fill="#C2A813"
+                  className="run-circle"
+                />
+                <circle
+                  cx="100"
+                  cy="100"
+                  r="35"
+                  fill="#fff"
+                  className="run-circle"
+                />
+                <circle
+                  cx="100"
+                  cy="100"
+                  r="25"
+                  fill="#C2A813"
+                  className="run-circle"
+                />
+                <circle
+                  cx="100"
+                  cy="100"
+                  r="15"
+                  fill="#fff"
+                  className="run-circle"
+                />
+              </g>
+              <g
+                fill="#fff"
+                fontWeight="bold"
+                fontSize={26}
+                letterSpacing="0.2em"
+                className="run-text"
+              >
+                <text>
+                  <textPath xlinkHref="#runs-path" href="#runs-path">
+                    RUNS
+                  </textPath>
+                </text>
+                <text transform="rotate(90,100,100)">
+                  <textPath xlinkHref="#runs-path" href="#runs-path">
+                    RUNS
+                  </textPath>
+                </text>
+                <text transform="rotate(180,100,100)">
+                  <textPath xlinkHref="#runs-path" href="#runs-path">
+                    RUNS
+                  </textPath>
+                </text>
+                <text transform="rotate(270,100,100)">
+                  <textPath xlinkHref="#runs-path" href="#runs-path">
+                    RUNS
+                  </textPath>
+                </text>
+              </g>
+            </svg>
           </div>
         </div>
       </div>
     );
   }
 }
 
+class Hand extends React.Component {
+  render() {
+    const cards = [0, 1, 2, 3, 4].map(i => <Card key={i} index={i} />);
+    return (
+      <div className="jest-hand">
+        {cards}
+        <script src="landing.js" />
```

Comment:
```suggestion
        <script src="/landing.js" />
```
Cards are broken with i18n urls (https://deploy-preview-7566--jest-preview.netlify.com/ro/) because it looks for the script in the wrong place.
Thanks for doing this!